pkg/api/vminfo.go

```diff
@@ -285,20 +286,19 @@ type ScalingConfig struct {
 }
 
 func (c *ScalingConfig) Validate() error {
-	ec := &erc.Collector{}
+	es := &ers.Stack{}
 
 	// Check c.loadAverageFractionTarget is between 0 and 2. We don't
-	// *strictly* need the upper
-	// bound, but it's a good safety check.
-	erc.Whenf(ec, c.LoadAverageFractionTarget < 0.0, "%s must be set to value >= 0", ".loadAverageFractionTarget")
-	erc.Whenf(ec, c.LoadAverageFractionTarget >= 2.0, "%s must be set to value < 2 ", ".loadAverageFractionTarget")
+	// *strictly* need the upper bound, but it's a good safety check.
+	es.Push(ers.Whenf(c.LoadAverageFractionTarget < 0.0, "%s must be set to value >= 0", ".loadAverageFractionTarget"))
+	es.Push(ers.Whenf(c.LoadAverageFractionTarget >= 2.0, "%s must be set to value < 2 ", ".loadAverageFractionTarget"))
 
 	// Make sure c.MemoryUsageFractionTarget is between 0 and 1
-	erc.Whenf(ec, c.MemoryUsageFractionTarget < 0.0, "%s must be set to value >= 0", ".memoryUsageFractionTarget")
-	erc.Whenf(ec, c.MemoryUsageFractionTarget >= 1.0, "%s must be set to value < 1 ", ".memoryUsageFractionTarget")
+	es.Push(ers.Whenf(c.MemoryUsageFractionTarget < 0.0, "%s must be set to value >= 0", ".memoryUsageFractionTarget"))
+	es.Push(ers.Whenf(c.MemoryUsageFractionTarget >= 1.0, "%s must be set to value < 1 ", ".memoryUsageFractionTarget"))
 
 	// heads-up! some functions elsewhere depend on the concrete return type of this function.
```

Comment:
I'm pretty sure this isn't true, based on some poking around, and while the stack type moved from `erc` to `ers` it should all work the same. 

There's also now an `ers.AsStack` helper that has some strong "dwim" vibes for converting/casting an error to a stack, which would be useful if this were true!